src/datepicker.js

```diff
@@ -1282,6 +1282,9 @@ function oneHandler(e) {
 
     // Hide all other instances.
     hideOtherPickers(instance)
+  } else if (type === 'keydown' && e.keyCode === 9 && instance) {    
```

Comment:
This is great. A few things...

1. Can you use a top level variable to store the keycode? And use `e.which || e.keyCode` for compatibility.
2. Can you refactor the lines below this which also check for keyCodes? I'll leave comments on each where you can replace that logic with the variable.

Thanks for this!

Reply:
It won't let me leave comments on the other lines, but as of this commit they are lines 1293 and 1297.